roms_tools/setup/river_forcing.py

```diff
@@ -674,7 +675,24 @@ def to_yaml(self, filepath: Union[str, Path]) -> None:
             The path to the YAML file where the parameters will be saved.
         """
 
-        _to_yaml(self, filepath)
+        # Serialize object into dictionary
+        yaml_data = _to_dict(self)
+        # Convert indices format
+
+        indices_data = yaml_data["RiverForcing"]["indices"]
+        serialized_indices = {
+            "_convention": "eta_rho, xi_rho"
+        }  # Add convention metadata
+
+        for key, value in indices_data.items():
+            serialized_indices[key] = [
+                f"{tup[0]}, {tup[1]}" for tup in value
+            ]  # Comma-separated string
+
+        yaml_data["RiverForcing"]["indices"] = serialized_indices
+
+        # Write to YAML
+        _to_yaml(yaml_data, filepath)
```

Comment:
Consider renaming `_to_yaml` to `write_yaml` to avoid the extraneous comment. As-is, I expect to see a return value based on the naming pattern previously established (e.g. `to_dict` returns a dict)

Reply:
Done in af3d3bc.